pkg/providers/tinkerbell/validator_test.go

```diff
@@ -56,6 +60,77 @@ func TestValidateMinimumRequiredTinkerbellHardware_EtcdUnspecified(t *testing.T)
 	assert.NoError(t, validator.ValidateMinimumRequiredTinkerbellHardwareAvailable(clusterSpec))
 }
 
+func TestValidateTinkerbellConfig_ValidAuthorities(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	pbnj := tinkerbellmocks.NewMockProviderPbnjClient(ctrl)
+	tink := tinkerbellmocks.NewMockProviderTinkClient(ctrl)
+	net := networkutilsmocks.NewMockNetClient(ctrl)
+
+	tink.EXPECT().GetHardware(gomock.Any())
+
+	var hardware hardware.HardwareConfig
+	datacenter := newValidTinkerbellDatacenterConfig()
+
+	validator := tinkerbell.NewValidator(tink, net, hardware, pbnj)
+	err := validator.ValidateTinkerbellConfig(context.Background(), datacenter)
+
+	assert.NoError(t, err)
+}
+
+func TestValidateTinkerbellConfig_InvalidGrpcAuthority(t *testing.T) {
+	cases := map[string]string{
+		"Missing port":     "1.1.1.1",
+		"Missing hostname": ":44",
+	}
```

Comment:
should we add a case where the port is provided but invalid, like `1.1.1.1:abc` or `1.1.1.1:111111`